MzIdentML/MzidIdentifications.cs

```diff
@@ -241,7 +243,8 @@ public bool IsDecoy(int sirIndex, int siiIndex)
             {
                 try
                 {
-                    foreach (mzIdentML111.Generated.PeptideEvidenceRefType pe
+
```

Comment:
This class is pretty ugly with all the nested try-catch blocks. If I understand what's going on, each of the mzIdentML classes has the same properties:
```
        private readonly mzIdentML110.Generated.MzIdentMLType110 dd110;
        private readonly mzIdentML111.Generated.MzIdentMLType111 dd111;
        private readonly mzIdentML120.Generated.MzIdentMLType120 dd120;
```
Why not use inheritance to greatly simplify this class and improve our ability to maintain it? Bring them all into the same namespace and make an `abstract class` that they inherit with the same properties, so you can just store this information in one variable, rather than having to try-catch in the three.